pkg/syncer/syncer.go

```diff
@@ -167,9 +169,24 @@ func (r *SyncController) Reconcile(ctx context.Context, origReq ctrl.Request) (_
 			return DeleteHostObject(syncContext, pObj, "virtual object uid is different")
 		}
 
-		return r.syncer.Sync(syncContext, pObj, vObj)
+		return r.genericSyncer.Sync(syncContext, &synccontext.SyncEvent[client.Object]{
+			Type:   syncEventType,
+			Source: syncEventSource,
+
+			Virtual:    vObj,
+			VirtualOld: nil,
+
+			Host:    pObj,
+			HostOld: nil,
```

Comment:
this is because it's not implemented yet right?

Reply:
Yes exactly, I will remove that